.changeset/early-knives-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+`Copy Button` - Fixed issue preventing copying of empty string and zero number values.
```

Comment:
[question] we're changing the `hds-clipboard` modifier, which is used also in the `Copy::Snippet` component; should this mentioned too in the changeset?

Reply:
Edited